frontend/TvCv19.Frontend/ClientApp/projects/patient/src/lib/patient/patient-registration/patient-registration.component.ts

```diff
@@ -6,30 +6,38 @@ import { PatientModel, PatientService } from 'projects/shared/src/public-api';
 @Component({
   selector: 'patient-registration',
   templateUrl: './patient-registration.component.html',
-  styleUrls: ['./patient-registration.component.scss']
+  styleUrls: ['./patient-registration.component.scss'],
 })
 export class PatientRegistrationComponent implements OnInit {
   patientRegistrationForm;
-  patients: Array<PatientModel> = []
-  constructor(private formBuilder: FormBuilder, private service: PatientService, private router: Router) {
+  patients: Array<PatientModel> = [];
+  constructor(
+    private formBuilder: FormBuilder,
+    private service: PatientService,
+    private router: Router
+  ) {
     this.patientRegistrationForm = this.formBuilder.group({
       name: '',
       location: '',
-      caregiverId: ''
-    })
+      age: '',
+      caregiverId: '',
+    });
   }
 
-  ngOnInit(): void {
-  }
+  ngOnInit(): void {}
   logPatients() {
```

Comment:
i know you didnt add this but can you delete as not being used

Reply:
ok deleted